src/data/cloud-computing-quiz.ts

```diff
@@ -1,4 +1,13 @@
 const cloudComputingQuiz = [
+  {
+    Question: "What is the cloud computing strategy often referred to as 'Lift and Shift'?",
+    Answer: "Rehosting",
+    Distractor1: "Repurchase",
+    Distractor2: "Replatform",
+    Distractor3: "Rearchitect",
+    Explanation: "Rehosting strategy is called “Lift and Shift” as you just move from point A to point B without making any changes. The advantage of this strategy is that it has a lower risk when migrating to the cloud",
```

Comment:
```suggestion
    Explanation: "The Rehosting strategy is often referred to as 'Lift and Shift' because you are moving your existing data and systems to the cloud with no changes. The advantage of this strategy is that it has a lower risk when migrating to the cloud",
```